app/Helpers/LinkHelper.php

```diff
@@ -86,7 +91,9 @@ static public function longLinkExists($long_url, $username=false) {
 
     static public function validateEnding($link_ending) {
         $is_valid_ending = preg_match('/^[a-zA-Z0-9-_]+$/', $link_ending);
-        return $is_valid_ending;
+        $is_not_restricted = ! in_array($link_ending, self::RESTRICTED_SLUGS);
```

Comment:
This makes sense, but would it be possible to use `\Validator` here as well in order to reduce inconsistency between this function and the validator in `LinkController`? 

https://laravel.com/docs/5.5/validation#manually-creating-validators

We could store the rules (i.e the array being passed to `$this->validate` in `LinkController`) in `LinkHelper`, then reference them from both `LinkHelper` and `LinkController`. Does that make sense?

Reply:
Yep, will update the PR.